jsonapi/fixtures_test.go

```diff
@@ -26,22 +26,35 @@ func (m MagicID) String() string {
 
 type Comment struct {
 	ID               int       `json:"-"`
+	LID              int       `json:"-"`
 	Text             string    `json:"text"`
 	SubComments      []Comment `json:"-"`
 	SubCommentsEmpty bool      `json:"-"`
 }
 
-func (c Comment) GetID() string {
-	return fmt.Sprintf("%d", c.ID)
+func (c Comment) GetID() Identifier {
+	id := Identifier{ID: fmt.Sprintf("%d", c.ID), Name: "comments"}
```

Comment:
No need to define Name, since GetName was not implemented before

Reply:
removed